keanu-python/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(name='keanu',
+      version='0.1',
+      author='Improbable Worlds',
+      author_email='UNKNOWN',
```

Comment:
@gordoncaleb What should the author's email be?

Reply:
`keanu-engineering@improbable.io`